commit/chainfee/types.go

```diff
@@ -1,16 +1,35 @@
 package chainfee
 
 import (
+	"time"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
 	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+	"github.com/smartcontractkit/chainlink-ccip/internal/plugincommon"
 )
 
 type Query struct {
 }
 
 type Outcome struct {
+	// Each Gas Price is the combination of Execution and DataAvailability Fees using bitwise operations
 	GasPrices []cciptypes.GasPriceChain `json:"gasPrices"`
 }
 
 type Observation struct {
-	GasPrices []cciptypes.GasPriceChain `json:"gasPrices"`
+	FeeComponents     map[cciptypes.ChainSelector]types.ChainFeeComponents    `json:"feeComponents"`
+	NativeTokenPrices map[cciptypes.ChainSelector]cciptypes.BigInt            `json:"nativeTokenPrice"`
+	FChain            map[cciptypes.ChainSelector]int                         `json:"fChain"`
+	ChainFeeUpdates   map[cciptypes.ChainSelector]plugincommon.ChainFeeUpdate `json:"chainFeeUpdates"`
+	Timestamp         time.Time                                               `json:"timestamp"`
```

Comment:
```suggestion
	TimestampNow         time.Time                                               `json:"timestamp"`
```

This naming feels more descriptive, `timestamp` can mean literally anything related to time.